src/Persistence/Sql.php

```diff
@@ -30,6 +30,9 @@ class Sql extends Persistence
     /** @const string */
     public const HOOK_BEFORE_DELETE_QUERY = self::class . '@beforeDeleteQuery';
 
+    /** @const string */
+    protected const ID_LOAD_ANY = self::class . '@idLoadAny';
```

Comment:
better constant than `null` to prevent typos/errors (like unassigned variable...)

Reply:
maybe we should support `one` (with checked only one record match) and `any` (one record of not strictly one)